src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -159,12 +157,26 @@ protected AbstractBuilding(@NotNull final IColony colony, final BlockPos pos)
     @Override
     public boolean hasModule(final Class<? extends IBuildingModule> clazz)
     {
-        return getFirstModuleOccurance(clazz).isPresent();
+        return getFirstModuleOccurance(clazz) != null;
```

Comment:
This is bad style on a `@NotNull` method.  It works, but it will annoy the code analysis.

Reply:
Yeah, I know, but it's more lightweight than wrapping it before that in an optional to do an isPresent check. So I'm fine to live with this.